### ISSUE TITLE
Non Qt-Creator build steps

### ISSUE THREAD

**Issue by mnadareski** [meepingsnesroms/libretro-palmm515emu#6]
It might be helpful to add the build steps for those people who don't want to install Qt-Creator in order to build from master. I'm not exactly proficient at this stuff, but I believe the steps are just:

```
cd qtBuildSystem/Mu
qmake
make
```

**Comment by parkerlreed** [comment]
For some odd reason, when I build it that way, stylus taps do not work.

Running through qtcreator, they do.

**Comment by meepingsnesroms** [comment]
What is there behavior when run without Qt creator, crash, do nothing, lock up the emulator?

Also did you touch the screen before the device booted up, that is almost a guaranteed lock up right now.

**Comment by parkerlreed** [comment]
Nope. Non-Qtcreator doesn't crash. The stylus calibration screen comes up (and animates) but clicking does nothing.

Qtcreator runs fine (If built using Qtcreator first. Building using qmake and then trying Qtcreator has the same result)

**Comment by parkerlreed** [comment]
Ok now both Qtcreator and qmake are showing the same behavior for me now. Can't click in the emulated screen.

**Comment by meepingsnesroms** [comment]
I dont know whats wrong in this case then, I will try to fix this after the chipselects detect 16mb and I load my test program to get real touchscreen data sent instead of writing to OS memory.

If it still happens when I get real touch data sent I will look into this further.

As for now I will open it as a separate issue for this so this one can be closed when the creator verifys its fixed.

**Comment by mnadareski** [comment]
Thanks for adding those tidbits to the main readme! 👍
